docs/observability/frontend/README.md

```diff
@@ -116,6 +116,28 @@ context.with(trace.setSpan(context.active(), span), () => {
 
 ```
 
+## Nais integrations
+
+The nais.yaml file has a field that allows for configuring the telemetry collector url automatically.
+If the `spec.frontend.generatedConfig.mountPath` is set you will get two things.
+
+A file at the specified path in your pod file system that contains the appropriate telemetry endpoint url configuration
+
+```js
+// mountPath: /src/app/static/example.js
+
+const vars = {
+	telemetryCollectorURL: '%s';
+};
+export default {vars};
+```
+this is the only value and it will point to the correct url depending on the environment.
+if you depend in this you should exclude it from your build system.
```

Comment:
```suggestion
This is the only value and it will point to the correct url depending on the environment.
If you depend on this you should exclude it from your build system.
```
Rettet bittelitt på språket.
Jeg skjønner ikke hva siste linje skal bety, men kanskje de som driver med frontend gjør det. Hvilken fil skal man eksludere fra byggesystemet sitt? Filen som blir mounted inn?